plugins/ipam/dhcp/lease.go

```diff
@@ -344,6 +360,45 @@ func (l *DHCPLease) maintain() {
 	}
 }
 
+func (l *DHCPLease) checkLinkExistsWithBackoff(ctx context.Context) (bool, error) {
+	checkFunc := func() (bool, error) {
+		_, err := netlink.LinkByName(l.link.Attrs().Name)
```

Comment:
If we assume that this link is already removed, then `l.link.Attrs()` may be invalid. We should store interface name ahead.